contracts/PartnerConfiguration/PartnerConfiguration.sol

```diff
@@ -161,17 +180,32 @@ contract PartnerConfiguration is IPartnerConfiguration, Ownable {
         uint256 /* expires */,
         uint256 duration
     ) external view override returns (uint256) {
-        require(
-            (duration >= 1) && (duration >= _minDuration),
-            "PartnerConfiguration: Less than min duration"
-        );
-
-        if ((_maxDuration != 0) && (duration > _maxDuration))
-            revert("PartnerConfiguration: More than max duration");
-
         if (duration == 1) return 2 * (10 ** 18);
         if (duration == 2) return 4 * (10 ** 18);
 
         return (duration + 2) * (10 ** 18);
     }
+
+    /**
+       @inheritdoc IPartnerConfiguration
+     */
+    function isValidName(string calldata name, uint256 duration) external view {
+        if (duration < _minDuration)
```

Comment:
**'<'** or **'<='**?? exclusive or inclusive?? I would say inclusive

Reply:
Well, I would say `<`, if we set the `minDuration` to be `3` for example, I would assume the `minDuration` can only be 3 and above.

Reply:
yes. you are right.